oshmem/mca/spml/ucx/spml_ucx.h

```diff
@@ -101,33 +101,48 @@ typedef struct mca_spml_ucx_ctx_array {
     mca_spml_ucx_ctx_t       **ctxs;
 } mca_spml_ucx_ctx_array_t;
 
+typedef struct mca_spml_ucx_rkey {
+    ucp_rkey_h rkey;
+    uint16_t refcnt;
```

Comment:
1. indent on column
2. IMO recent better be int, to avoid surprises

Reply:
fixed